### PR TITLE
[W5.11][F10-1]Mick Lee Wei Lin

### PR DESCRIPTION
**Pull request by micklwl**
Ready for review

### CODE REVIEW

**Review by 0WN463 — COMMENTED** [review]
Just a nitpick, 
For the imports, you used wild card, most likely due to the IDE doing that itself
"import java.util.*;"
The coding format guide advised us not to do so, but rather import each module individually.

File: src/seedu/addressbook/common/Messages.java [hunk 1]
```diff
@@ -11,6 +11,7 @@
     public static final String MESSAGE_INVALID_PERSON_DISPLAYED_INDEX = "The person index provided is invalid";
     public static final String MESSAGE_PERSON_NOT_IN_ADDRESSBOOK = "Person could not be found in address book";
     public static final String MESSAGE_PERSONS_LISTED_OVERVIEW = "%1$d persons listed!";
+    public static final String MESSAGE_PERSONS_SORTED_OVERVIEW = "%1$d persons sorted!";
```
Comment (stephlewyh) [inline src/seedu/addressbook/common/Messages.java:14]: Nice statistic to have.

File: src/seedu/addressbook/data/AddressBook.java [hunk 2]
```diff
@@ -68,6 +68,11 @@ public UniquePersonList getAllPersons() {
         return new UniquePersonList(allPersons);
     }
 
+    /**
+     * Sorts all persons in the address book.
+     */
+    public void sortPersons(){ allPersons.sort(); }
```
Comment (stephlewyh) [inline src/seedu/addressbook/data/AddressBook.java:74]: Please adhere to coding standard for methods.

```
public void sortPersons() {
    allPersons.sort();
}
```

**Review by stephlewyh — COMMENTED** [review]
Hi Mick, well done! 
(1) Use of OOP standard in code
(2) Provided appropriate JUnit test and test cases. 
(3) Updated user doc.

You may close this PR after reviewing our comments.

**Pull request closed without merging**